todo_list/views.py

```diff
@@ -1,3 +1,88 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
 
-# Create your views here.
+from todo_list.forms import PageForm, TaskForm
+from todo_list.models import Page, Task
+
+
+def dashboard_view(request):
+    return render(request, 'dashboard.html')
+
+
+# Page Views #
+
+def list_page_view(request):
+    page_list = Page.objects.all()
+    task_list = Task.objects.all()
+
+    context = {'page_list': page_list, 'task_list': task_list}
+    return render(request, 'todo_list/page/page_list.html', context)
+
+
+def create_page_view(request):
+    if request.method == 'GET':
+        form = PageForm()
+        return render(request, 'todo_list/page/page_create.html', {'form': form})
+
+    else:
+        form = PageForm(request.POST)
+        if form.is_valid():
```

Comment:
@quratulain25 You should return response with errors in case form is invalid. And do this for all views.

Reply:
@ifzal , done.